src/NServiceBus.NHibernate/TimeoutPersisters/NHibernateTimeoutStorage.cs

```diff
@@ -54,10 +56,14 @@ protected override void Setup(FeatureConfigurationContext context)
                 }
             }
 
+            var timeoutsCleanupExecutionInterval = 
+                context.Settings.GetOrDefault<TimeSpan?>("NHibernate.Timeouts.CleanupExecutionInterval") ?? TimeSpan.FromMinutes(2);
```

Comment:
I'm not sure if we want to give any guarantees. I think the only thing we say about deferred messages is that they won't get delivered earlier than. @andreasohlund thoughts?